core/examples/luigi-sample-angular/src/app/project/project.component.ts

```diff
@@ -7,13 +7,12 @@ import {
 } from '@angular/core';
 import { ActivatedRoute, Params } from '@angular/router';
 import { Subscription } from 'rxjs';
-
 import {
   linkManager,
   uxManager,
   addContextUpdateListener,
   removeContextUpdateListener
-} from '@kyma-project/luigi-client';
+} from '@kyma-project/luigi-client/luigi-client-legacy';
```

Comment:
I'd like to see `@kyma-project/luigi-client/legacy` but it's kind of odd to create a copy of the bundle just for this.

Reply:
I agree